lib/oli_web/live/products/details/content.ex

```diff
@@ -3,43 +3,73 @@ defmodule OliWeb.Products.Details.Content do
   alias Surface.Components.Link
   alias OliWeb.Router.Helpers, as: Routes
 
+  import Ecto.Changeset
```

Comment:
Nit but we try to group all the `use`, `import` and `alias` together by type, and I think the convention says that's the order too.